src/commands/createNewProject/pythonSteps/addPythonCreateProjectSteps.ts

```diff
@@ -3,24 +3,21 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AzureWizardExecuteStep, AzureWizardPromptStep } from "vscode-azureextensionui";
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
 import { getWorkspaceSetting } from "../../../vsCodeConfig/settings";
 import { IProjectWizardContext } from '../IProjectWizardContext';
 import { PythonProjectCreateStep } from "../ProjectCreateStep/PythonProjectCreateStep";
 import { IPythonVenvWizardContext } from "./IPythonVenvWizardContext";
-import { PythonAliasListStep } from "./PythonAliasListStep";
 import { PythonVenvCreateStep } from "./PythonVenvCreateStep";
 
 export function addPythonCreateProjectSteps(
     context: IProjectWizardContext & IPythonVenvWizardContext,
-    promptSteps: AzureWizardPromptStep<IProjectWizardContext>[],
     executeSteps: AzureWizardExecuteStep<IProjectWizardContext>[]): void {
 
     const createPythonVenv: boolean = !!getWorkspaceSetting<boolean>('createPythonVenv', context.workspacePath);
     context.telemetry.properties.createPythonVenv = String(createPythonVenv);
 
     if (createPythonVenv) {
-        promptSteps.push(new PythonAliasListStep());
         executeSteps.push(new PythonVenvCreateStep());
```

Comment:
We don't need to add this step either since it's added in `addPythonInitVSCodeSteps.ts` now. In fact, there's only one line from this file that's actually needed, so I would delete this file and just add this line directly to `NewProjectLanguageStep.ts`:

```
executeSteps.push(new PythonProjectCreateStep());
```

Reply:
How about this line? Isn't it needed for the telemetry?
```context.telemetry.properties.createPythonVenv = String(createPythonVenv);```

Reply:
How about this line? Isn't it needed for the telemetry?
context.telemetry.properties.createPythonVenv = String(createPythonVenv);

Reply:
Not if you move it to the other file like my other comment suggests

Reply:
Sorry. Missed your other comment. I have now moved the code from `addPythonInitVSCodeSteps.ts` into `NewProjectLanguageStep.ts` as suggested.